src/main/scala/goatrodeo/omnibor/Storage.scala

```diff
@@ -248,7 +149,7 @@ object StorageReader {
     if (url.getProtocol() == "file") {
       new FileStorageReader(url.getPath())
     } else {
-      new WebStorageReader(url)
+      throw new Exception(f"Cannot build storage reader for ${url}")
```

Comment:
This should be a custom Exception that both declares what it is and allows fine grained catches.

Reply:
Will fix